### ISSUE TITLE
MMU2s extruder doesn't always release filament when loading to hotend completes

### ISSUE THREAD

**Issue by Brandoskey** [prusa3d/Prusa-Firmware#1573]
This is on an mk3s/mmu2s 3.6.0 and 1.0.5 respectively with a factory reset

MM prints will cycle through the first 5 filament changes fine, then when getting back to filament 1 it will load the filament to the hotend, trigger the sensor (verified through the sensors menu) and normally you'd expect the extruder motor on the mmu2 to disengage the filament and allow it to move freely, instead it stays engaged causing the hotend extruder to start grinding the filament, eventually the filament is so ground down the IR sensor will read 0, which perhaps would be a way to tell the einsy that something has gone terribly wrong (feature request?)

After not printing this filament the mmu2 will move on to the next filament and works as expected, generally continuing to do so until it gets back to filament 1.

**Comment by Brandoskey** [comment]
After more testing, the filament number it fails on seems more random now.

Additional steps I've taken are to 

-check for anything that might be keeping the idler from moving freely
-tried different combinations of firmware including 3.5.3 and 3.6.0 for the mk3s and 1.0.4 and 1.0.5 for the mmu2
-factory resets for the mmu2 and mk3s
-reflashing the firmwares
-printing pre sliced gcode from sample gcodes
-slicing with the latest version of slic3r pe using the mk3s mmu2s profiles

**Comment by Brandoskey** [comment]
After more testing and another factory reset of the mmu2 I believe this may not be a bug.

I was given the impression by support that even with the new sensor you still needed to do the load distance calibration and I think the fact that I did that calibration was giving me my issues. After the final factory reset, I skipped the calibration and everything worked perfectly. Based on how the new sensor works, this makes sense to me.

Printed a 5 color gear bearing with 300 tool changes and never had a single problem.

I'll close this
